src/test/clojure/xtdb/pgwire_test.clj

```diff
@@ -2105,6 +2103,29 @@ ORDER BY t.oid DESC LIMIT 1"
         (with-open [rs (.executeQuery stmt)]
           (t/is (= [{"arr" [1 2 3]}] (rs->maps rs))))))))
 
+(deftest test-text-array
```

Comment:
test for reading the text array? assume we should be able to pass one in as a parameter